src/components/modal/AddModal.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import {
+  AddModalWrapper,
+  BlueButton,
+} from '../../styles/styledComponents/folderStyled';
+import { AddModalButton } from './AddModalButton';
+
+export const AddModal = ({ modalData }) => {
+  const overScroll = modalData.folderName.length > 4 ? 'true' : 'false';
+  const linkCounts = modalData.linkCounts;
+  const [addModalCheck, setAddModalCheck] = useState(-1);
+  const handleAddModalCheck = (e, value) => {
+    e.preventDefault();
+    setAddModalCheck(value);
+  };
+
+  // 링크 입력창 버튼
+  return (
+    <>
+      <AddModalWrapper $overscroll={overScroll}>
+        {modalData.folderName.map((folder, idx) =>
+          addModalCheck === idx ? (
+            <AddModalButton
+              check
+              folder={folder}
+              idx={idx}
```

Comment:
```suggestion
              index={index}
```
변수명을 단축하는건 코드 가독성을 떨어뜨립니다.
지양하시는걸 권장드려요!